### PR TITLE
🎍 Fix alepp

### PR DESCRIPTION
**Pull request by MaciejNadolski98**

### CODE REVIEW

File: src/gradient.py [hunk 1]
```diff
@@ -131,45 +130,53 @@ def fool_acc(self, max_iter=None, verbose=True, grid=None,random_state=None, met
...
         y = tf.convert_to_tensor(self._y, dtype=tf.float32)
-        loss = self.calculate_alepp_loss(self._X, y, target, reg_factor)
+        raw_loss, loss = self.calculate_alepp_loss(self._X, self._X_changed, y, reg_factor)
         self.iter_losses['iter'].append(0)
+        self.iter_losses['raw_loss'] = [raw_loss]
         self.iter_losses['loss'].append(loss)
 
         self.explainer.model.evaluate(self._X, self._y)
-        optimizer = tf.keras.optimizers.Adam(learning_rate=self.params['learning_rate'])
+        optimizer = tf.keras.optimizers.Adam()
```
Comment (crewtool) [inline src/gradient.py:143]: Czemu wywalony learning rate?

File: bike-sharing.py [hunk 2]
```diff
@@ -80,25 +80,24 @@
...
 alg.plot_explanation(method=args.method, title=title)
-alg.plot_data(constant=False)
+# alg.plot_data(constant=False)
 
 #----------------------- ale++ --------------------------
 
 X_changed = pd.DataFrame(data=alg._X_changed, columns=X.columns)
-explainer_alepp = src.Explainer(alg.explainer.model, X_changed, y)
+orig_explainer = src.Explainer(alg.explainer.model, X, y)
+changed_explainer = src.Explainer(alg.explainer.model, X_changed, y)
```
Comment (crewtool) [inline bike-sharing.py:92]: Never used

File: src/gradient.py (hunk 1 above)
Comment (MaciejNadolski98) [reply src/gradient.py:143]: Z tego co pamiętam to jest learning rate do innego procesu uczenia; możliwe że to do trenowania ataku, podczas gdy tutaj trenujemy sam model.
